app/poros/api_obituary.rb

```diff
@@ -1,15 +1,21 @@
+require "date"
+
 class ApiObituary
   attr_reader :headline,
               :abstract,
               :web_url,
               :pub_date,
               :word_count
-              
+
   def initialize(info)
     @headline = info[:data][:attributes][:headline]
     @abstract = info[:data][:attributes][:abstract]
     @web_url = info[:data][:attributes][:web_url]
-    @pub_date = info[:data][:attributes][:pub_date]
+    @pub_date = format_date(info[:data][:attributes][:pub_date])
     @word_count = info[:data][:attributes][:word_count]
   end
+
+  def format_date(date)
+    date.split("T")[0].to_date.strftime("%B %-d, %Y")
```

Comment:
the date that comes back from NYT is pretty funky, but this cleaned it up into a nice little month day, year format.